jopa-impl/src/main/java/cz/cvut/kbss/jopa/oom/ObjectOntologyMapperImpl.java

```diff
@@ -264,7 +267,18 @@ public <T> T getEntityFromCacheOrOntology(Class<T> cls, URI identifier, Descript
             // This prevents endless cycles in bidirectional relationships
             return cls.cast(existing);
         } else {
-            return loadEntityInternal(new LoadingParameters<>(cls, identifier, descriptor));
+            // setup loading params
+            LoadingParameters<T> params = new LoadingParameters<>(cls, identifier, descriptor);
+
+            // TODO:
```

Comment:
I agree with the comment. Please implement the proposed solution - propagation of LoadingParameters (or their relevant part) to this method.